src/components/seeds/database/index.tsx

```diff
@@ -565,7 +565,7 @@ export default function Database(props: DatabaseProps): JSX.Element {
           />
         </>
       )}
-      <TfMain>
+      <TfMain backgroundImageVisible={!isOnboarded}>
```

Comment:
@joshbuchea the screenshots in the PR seem to indicate otherwise. Do they need to be updated?

Reply:
@karthikbtf I _think_ the screenshots accurately reflect the code change 🤔  The background image should be visible/shown, when the user hasn't completed onboarding...

Reply:
Oh unless you're referring to the background for the `NavBar`. In that case, there's a note in the PR description.

Reply:
ugh yes my bad, was looking at the navbar transparency - totally unrelated

Reply:
sorta related but unrelated to this PR

Reply:
It's all good, it's related. I appreciate your help with thorough PR reviews & testing. Thanks! 🙌